superset/migrations/versions/453530256cea_add_save_option_column_to_db_model.py

```diff
@@ -0,0 +1,48 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+"""add_save_form_column_to_db_model
+
+Revision ID: 453530256cea
+Revises: f1410ed7ec95
+Create Date: 2021-04-30 10:55:07.009994
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = "453530256cea"
+down_revision = "f1410ed7ec95"
+
+import sqlalchemy as sa
+from alembic import op
+
+from superset.models.core import ConfigurationMethod
```

Comment:
Let's hard code this to ensure the migration script doesn't get broken if someone changes the enum in the future.

Reply:
ok, I wasn't sure what method would be preferable, to hard code or to import, looking through the versions I saw a little bit of both.